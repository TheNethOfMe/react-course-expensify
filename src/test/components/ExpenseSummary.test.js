import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary.jsx';

test('should render with one expense viewed correctly', () => {
  const wrapper = shallow(<ExpenseSummary expensesTotal={195} expenseCount={1} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render with multiple expenses viewed correctly', () => {
  const wrapper = shallow(<ExpenseSummary expensesTotal={114195} expenseCount={3} />);
  expect(wrapper).toMatchSnapshot();
});