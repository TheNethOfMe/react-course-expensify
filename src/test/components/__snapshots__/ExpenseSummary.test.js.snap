// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with multiple expenses viewed correctly 1`] = `
<div>
  <p>
    Viewing 
    3
     
    expenses
     totalling 
    $1,141.95
    .
  </p>
</div>
`;

exports[`should render with one expense viewed correctly 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
     totalling 
    $1.95
    .
  </p>
</div>
`;
